Dockerfile

```
@@ -295,7 +295,8 @@ RUN ./configure $GRASS_CONFIG \
     mv module_items.xml /usr/local/grass85/gui/wxpython/xml/module_items.xml;
 
 # Build the GDAL-GRASS plugin
-RUN git clone https://github.com/OSGeo/gdal-grass \
+ARG GDAL_GRASS_VERSION=1.0.3
+RUN git clone --branch $GDAL_GRASS_VERSION --single-branch https://github.com/OSGeo/gdal-grass.git \
```

`--depth 1` is even faster than `--single-branch`:

```suggestion
RUN git clone --branch $GDAL_GRASS_VERSION --depth 1 https://github.com/OSGeo/gdal-grass.git \
```

---

Maybe we save some micro secs for that giga-repo :-) , updated.

---

True, but there's only 109 commits for now, so in this specific case we probably won't notice it very much.